Apps/WebClient/src/featuretoggleconfig.json

```diff
@@ -87,7 +80,7 @@
                 "enabled": false
             },
             {
-                "name": "specialAuthority",
+                "name": "specialAuthorityRequest",
```

Comment:
I don't remember if this change also needs to happen in the functional tests or not, and in the related EntryType